packages/strapi-plugin-content-manager/admin/src/components/State/index.js

```diff
@@ -5,11 +5,11 @@ import { Text } from '@buffetjs/core';
 import { getTrad } from '../../utils';
 import Wrapper from './Wrapper';
 
-const State = ({ isPublished }) => {
+const State = ({ isPublished, style }) => {
   const { formatMessage } = useIntl();
 
   return (
-    <Wrapper isGreen={isPublished}>
+    <Wrapper isGreen={isPublished} style={style}>
```

Comment:
You can remove the style props here and add lineHeight 19 to the Text component below.